### PR TITLE
Update networking.md

### PR DESCRIPTION
**Pull request by stkasulanati**
Replaced the word "between" with among, as there can more than two applications sharing the same machines.

### CODE REVIEW

**Review by kundan2707 — COMMENTED** [review]
/lgtm

**Comment by reylejano** [comment]
This is a trivial edit, please see our policy on trivial edits
https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits

I'm neutral on this PR, I don't feel that "between applications" implies 2 applications

File: content/en/docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ problems to address:
 
 <!-- body -->
 
-Kubernetes is all about sharing machines between applications.  Typically,
+Kubernetes is all about sharing machines among applications.  Typically,
 sharing machines requires ensuring that two applications do not try to use the
 same ports.  Coordinating ports across multiple developers is very difficult to
 do at scale and exposes users to cluster-level issues outside of their control.
```
Comment (ricardoamaro) [inline content/en/docs/concepts/cluster-administration/networking.md:25]:
```suggestion
do at scale and exposes users to cluster-level issues outside their control.
```

**Pull request merged**